src/measure_theory/measure/stieltjes.lean

```diff
@@ -352,6 +352,65 @@ begin
       measure_union A measurable_set_Ioo, f.mono.le_left_lim hab, ← ennreal.of_real_add] }
 end
 
+lemma measure_Iic {l : ℝ} (hf : tendsto f at_bot (𝓝 l)) (x : ℝ) :
+  f.measure (Iic x) = of_real (f x - l) :=
+begin
+  have h_tendsto_1 : tendsto (λ r : ℚ, f.measure (Ioc r x)) at_bot (𝓝 (f.measure (Iic x))),
+  { have h_Iic_eq_Union : Iic x = ⋃ r : ℚ, Ioc (↑-r) x,
+    { ext1 x,
+      simp only [mem_Iic, mem_Union, mem_Ioc, exists_and_distrib_right, iff_and_self],
+      intro h,
+      simp_rw [rat.cast_neg, neg_lt],
+      exact exists_rat_gt _, },
+    rw h_Iic_eq_Union,
+    suffices h_neg_top : tendsto (λ r : ℚ, f.measure (Ioc (↑-r) x)) at_top
+      (𝓝 (f.measure (⋃ r : ℚ, Ioc (↑-r) x))),
+    { have : (λ (r : ℚ), f.measure (Ioc (↑r) x)) = (λ r, f.measure (Ioc (↑- -r) x)),
+      { simp_rw neg_neg, },
+      rw this,
+      exact h_neg_top.comp tendsto_neg_at_bot_at_top, },
+    refine tendsto_measure_Union (λ r r' hrr' x, _),
+    simp only [rat.cast_neg, mem_Ioc, and_imp],
+    refine λ hrx hxq, ⟨(neg_le_neg _).trans_lt hrx, hxq⟩,
+    exact_mod_cast hrr', },
+  have h_tendsto_2 : tendsto (λ r : ℚ, f.measure (Ioc r x)) at_bot (𝓝 (of_real (f x - l))),
+  { simp_rw measure_Ioc,
+    refine ennreal.tendsto_of_real (tendsto.const_sub _ (hf.comp _)),
+    rw tendsto_coe_rat_at_bot_iff,
+    exact tendsto_id, },
+  exact tendsto_nhds_unique h_tendsto_1 h_tendsto_2,
+end
+
+lemma measure_Ici {l : ℝ} (hf : tendsto f at_top (𝓝 l)) (x : ℝ) :
+  f.measure (Ici x) = of_real (l - left_lim f x) :=
+begin
+  have h_tendsto_1 : tendsto (λ r : ℚ, f.measure (Ico x r)) at_top (𝓝 (f.measure (Ici x))),
```

Comment:
This fact would deserve its own lemma. But it would be better to prove it for any measure on the reals, not only Stieltjes measures. And also the result is true along reals, not just rationals, I think? Also, if instead of the reals you prove it in any second countable linear order, then I think you can deduce the result at -infinity from the result at +infinity by applying it to the order-dual space?

Reply:
Thanks for that review. I isolated that lemma (and similar ones). They are still in the same file but I marked them to be moved after the port.
I could not make order_dual work for `tendsto_measure_Ioc_at_bot` (but it worked for `tendsto_measure_Ici_at_bot`).